pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
```

Comment:
Great work on this PR! 
I noticed that `wheel` is included as a build dependency. While it’s definitely helpful for distribution purposes, I was wondering if it’s something we need at this stage of the project.
 @peterdudfield, could you share your thoughts on whether `wheel` is necessary for now or if it’s more of a forward-looking addition?

Reply:
Yea, I'd probably leave wheel out for the moment

Reply:
Alright thank you for your input @peterdudfield . @siddharth7113 work on top of this PR or should i make the necessay changes .

Reply:
@siddharth7113 sure !

Reply:
@peterdudfield , removed the build dependency from the pyproject.toml file